cypress/e2e/finance/funds/test-acquisition-unit-restrictions-for-funds.cy.js

```diff
@@ -61,20 +61,21 @@ describe('ui-finance: Funds', () => {
   });
 
   after(() => {
-    cy.loginAsAdmin({ path:SettingsMenu.acquisitionUnitsPath, waiter: AcquisitionUnits.waitLoading });
-    AcquisitionUnits.unAssignAdmin(defaultAcquisitionUnit.name);
-    AcquisitionUnits.delete(defaultAcquisitionUnit.name);
-    cy.visit(TopMenu.fundPath);
+    cy.loginAsAdmin({ path:TopMenu.fundPath, waiter: Funds.waitLoading });
     FinanceHelp.searchByName(defaultfund.name);
     FinanceHelp.selectFromResultsList();
     Funds.deleteFundViaActions();
```

Comment:
please explain why you need this wait

Reply:
line 68

Reply:
Robot sometimes manages to delete everything, but it happens that the test falls as not enough time to clean the data of fund.on this I decided to put cy.wait(), so he just had time to work out.

Reply:
write a comment wait until the message dissappeared, wait until object/element dissapear. you can use in this case the cy.expect to something not exist